### PR TITLE
Collect errors from transformers

### LINKED ISSUE #765: private input with no default value - no inputs return at all!!

### PR DESCRIPTION
**Pull request by Bonczidai**
fixes #765

Signed-off-by: Bonczidai Levente <levente.bonczidai@hpe.com>

### CODE REVIEW

File: cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/NavigateTransformer.java [hunk 1]
```diff
@@ -25,36 +27,43 @@
...
+                        throw new RuntimeException("Each key in the navigate section should be a string.");
+                    }
+                    if (!(navigationValue instanceof String)) {
+                        throw new RuntimeException("Each value in the navigate section should be a string.");
+                    }
+                    @SuppressWarnings("unchecked")
+                    Map<String, String> elementAsStringMap = elementAsMap;
+                    transformedData.add(elementAsStringMap);
+                } else {
+                    throw new RuntimeException();
```
Comment (gituser173) [inline cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/NavigateTransformer.java]: no message for the RuntimeException?

File: cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/ParallelLoopForTransformer.java [hunk 2]
```diff
@@ -39,3 +40,4 @@ public String keyToTransform() {
     }
 
 }
+;
```
Comment (gituser173) [inline cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/ParallelLoopForTransformer.java]: remove this semicolon

File: cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/NavigateTransformer.java (hunk 1 above)
Comment (Bonczidai) [inline cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/NavigateTransformer.java]: I believe it is not recommended :bowtie:

File: cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/ParallelLoopForTransformer.java (hunk 2 above)
Comment (Bonczidai) [inline cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/ParallelLoopForTransformer.java]: :ok_hand:

**Comment by Bonczidai** [comment]
thanks @gituser173 for the review

**Pull request merged**